java/src/main/native/src/RmmJni.cpp

```diff
@@ -1106,4 +1106,15 @@ JNIEXPORT void JNICALL Java_ai_rapids_cudf_Rmm_freeFromFallbackPinnedPool(JNIEnv
   }
   CATCH_STD(env, )
 }
+
+JNIEXPORT void JNICALL Java_ai_rapids_cudf_Rmm_configureDefaultCudfPinnedPoolSize(JNIEnv* env,
+                                                                                  jclass clazz,
```

Comment:
Remove name of the unused var to avoid compile warning.
```suggestion
                                                                                  jclass,
```

Reply:
We have this in every single API in this file and probably other, but I am not seeing warnings on it. I'd rather keep it as is at this stage.